packages/shared/src/components/fields/MarkdownInput/index.tsx

```diff
@@ -262,7 +262,7 @@ function MarkdownInput(
         appendTo={parentSelector}
       />
       {footer ?? (
-        <span className="flex flex-row items-center justify-end gap-3 border-t border-theme-divider-tertiary p-3 px-4 text-theme-label-tertiary">
+        <span className="flex flex-row items-center gap-3 border-theme-divider-tertiary p-3 px-4 text-theme-label-tertiary laptop:justify-end laptop:border-t">
```

Comment:
style changes for mobile, where the footer is left aligned and does not have a divider border on top